charts/internal/calico/templates/node-cpva/deployment-calico-node-vertical-autoscaler.yaml

```diff
@@ -0,0 +1,55 @@
+---
+kind: Deployment
+apiVersion: {{ include "deploymentversion" . }}
+metadata:
+  name: calico-node-vertical-autoscaler
+  namespace: kube-system
+  labels:
+    origin: gardener
```

Comment:
The origin is `gardener-extension-networking-calico`, but I'm not sure if this was adapted everywhere else yet.

Reply:
btw, what is this origin field used for?

Reply:
Historically, it was used by the shoot care controller to identify the pods spawned by Gardener in the shoot in order to perform health checks only for them. Later on, more labels got introduced (role=system-component, for example), making this `origin=gardener` label kind of obsolete. From my point of view it can also be removed in favor of the newer labels.